src/app/components/IngredientsList/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import List from '@mui/material/List/List';
+import IngredientListItem from '../IngredientListItem';
+
+const initialState = [
+  { id: 1, amount: '300g', title: 'pasta' },
+  { id: 2, amount: '100g', title: 'pesto' },
+  { id: 3, amount: '30ml', title: 'olive oil' },
+];
+
+export default () => {
+  return (
+    <List
+      sx={{
+        width: '100%',
+        maxWidth: 360,
+        bgcolor: 'background.paper',
+        mt: '5px',
+        pr: '100px',
```

Comment:
I'd consider using relative sizes.

Reply:
Thanks for the suggestion! I won't do that here though because those are margin-top and padding-right. I don't want users who make their fonts bigger to have to scroll through margin before seeing any content :D